### ISSUE TITLE
Replicating Taco2+WaveRNN

### ISSUE THREAD

**Issue by fuchini** [mozilla/TTS#226]
Hi again, I'm currently trying to replicate your Taco2+WaveRNN results with the same dataset (LJSpeech) but I have a couple of questions:

1.  Do you start with batchnorm prenet or original and then finetune with bn? Is it still recommended to start with r=5 and the move to r=1?
2. How many GPU's are you using? How many epochs and batch_size?
3. How long does it take to train and to align?
4. Is it better to use sigmoid or softmax? I'm getting better results from sigmoid 

Thanks a lot

**Comment by erogol** [comment]
1. Just training with r=1 no change.
2. 2 gpus with 32 batch size in total
3. around 30K it needs to be aligned but sometimes it goes until 90K
4. I also prefer sigmoid. 

I guess config_cluster.json is the most suitable config file you can use for your training. It has all the hyperparameters set for you.

Good luck !
